compiler/runtime/OMRCodeCache.cpp

```diff
@@ -224,6 +224,7 @@ OMR::CodeCache::trimCodeMemoryAllocation(void *codeMemoryStart, size_t actualSiz
    uint8_t *expectedHeapAlloc = (uint8_t *) codeMemoryStart + oldSize;
    if (config.verboseReclamation())
       {
+      TR_VerboseLog::vlogAcquire();
```

Comment:
We can not acquire lock and then call `writeLineLocked`. `writeLineLocked` manages acquiring and releasing the verbose log.